packages/block-library/src/verse/block.json

```diff
@@ -10,14 +10,29 @@
 			"default": "",
 			"__unstablePreserveWhiteSpace": true
 		},
+		"style": {
```

Comment:
Setting defaults in this way will make sure they are included as inline styles. Doing that will make theme.json not work as expected because theme.json does not produce inline styles but targets using a class. Also including inline styles even if the user does not change anything does not seems like a good idea. As all the verse blocks would have this markup instead of just having a single class with the style rule we would repeat it every time.

Allowing blocks to set default style properties may be a good idea. An equivalent to theme.json that blocks can use to target their own selector. In the future, I guess we should probably consider this. But I guess for now these defaults should just be added as CSS to the style.scss file.

Reply:
This is how the docs say we should do it: https://developer.wordpress.org/block-editor/developers/block-api/block-supports/#spacing

Reply:
It seems the documentation refers to something that at least for now should not be used. I will propose a PR to remove that from the documentation. Thank you for bringing our attention to this documentation isse.